### PR TITLE
fix(curriculum): fix casing of Dense keyword in quiz

### PR DESCRIPTION
**Pull request by sidemt**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

## Issue:
In the quiz the code below, it is written as : `__A__.keras.layers.DENSE(1, activation='sigmoid')`
But as far as I checked, Python is case sensitive and the correct code should be `Dense` instead.

## Affected page:
https://www.freecodecamp.org/learn/machine-learning-with-python/tensorflow/natural-language-processing-with-rnns-sentiment-analysis

quiz

## References:
You can check the behavior with the Colab notebook provided for this video:
https://colab.research.google.com/drive/1ysEKrw_LE2jMndo1snrZUh5w87LQsCxk?usp=sharing

Keras API Reference: 
https://keras.io/api/layers/core_layers/dense/

### CODE REVIEW

**Review by ShaunSHamilton — APPROVED** [review]
Thank you, for opening this.

This LGTM 👍

**Review by gikf — APPROVED** [review]
LGTM :+1:
